### ISSUE TITLE
add raw $response to returned data structures

### ISSUE THREAD

**Issue by glortho** [DigitalGlobe/jetset#1]
right now, if you use `getData` to do something with the response from `$get()` or `$list()` you can't get to the original response

**Comment by g45t345rt** [comment]
That is kind of big issue.
You can't access response headers or any other response properties from within Jetset.

I think it's because of iso-fetch-stream returning response.json() correct me if I'm wrong
https://github.com/glortho/iso-fetch-stream/blob/26d12b77cf74e98b5fcb336d7ec3a4edd908c206/src/index.js#L55

What to do if I want to access number of results in the Content-Range response header? Hmm...
Rest Finale pagination: https://github.com/tommybananas/finale#pagination

**Comment by glortho** [comment]
You're right @g45t345rt -- it's set up only for simplest response handling
at the moment. But we'd be very happy to review a PR that improves upon
this limitation!

**Comment by g45t345rt** [comment]
This looks like a lot of work for just a simple thing :s 
I will think about it.

Thanks,
glortho
